.tugboat/config.yml

```
@@ -1,6 +1,6 @@
 services:
   php:
-    image: tugboatqa/php:8.1-apache-bullseye
+    image: tugboatqa/php:8.2-apache-bullseye
```

@indigoxela can you please try switching this to the image that doesn't specify the OS?:
```suggestion
    image: tugboatqa/php:8.2-apache
```

IIRC, we didn't have the same issues with that image as we were having with the `8.1-apache` one, so perhaps that could also fix https://github.com/backdrop/backdrop-issues/issues/6139 for us.

Besides, when we merged the change with the `8.1-apache-bullseye`, we said that we are explicitly doing that in order to solve the issue where the sandboxes were being provisioned with PHP 7.2, and that we should look to decouple us from having to specify the image at the OS version level.

---

@indigoxela scratch that ^^

Things should be fixed now (see https://github.com/backdrop/backdrop-issues/issues/6139), but you'll need to rebase. Can you please do that when you get a chance? Thanks 🙏🏼

---

That's great, we have a PR sandbox again! :tada: 

Rebased and things seem to work as expected. :+1:

---

@indigoxela actually can we please try to remove the OS suffix from the image? If that works, then we should use that, as @quicksketch has requested to not couple ourselves with OS-specific images ...we only had to do that for PHP 8.1, because the sandboxes were being provisioned with 7.2 for some reason, but that was simply a workaround for that specific problem - never meant to be a permanent change.

---

Sure, done. :+1: